tests/ci/spec_to_test_mapping.py

```diff
@@ -47,7 +47,7 @@
     'components/io-libs/hdf5/SPECS/hdf5.spec': [
         'hdf5',
         '',
-        'zlib-devel'
+        'zlib-devel automake-ohpc libtool-ohpc autoconf-ohpc'
```

Comment:
This line probably also needs to be added for phdf5 below

Reply:
Yes! Done.

Reply:
BTW, I didn't squash all commits into one, becuase I think it might help for future reference when someone want to change hdf5 test. : )